backend/hyperglosae/src/updates/document.js

```diff
@@ -0,0 +1,17 @@
+function(doc, req) {
+  const body = JSON.parse(req.body, (k, v) => (k === 'history') ? undefined : v);
+  if (!doc) {
+    doc = Object.assign({}, body, {
+      _id: req.id,
+      _rev: req.rev,
+      history: [{
+        user: req.userCtx.name,
+        date: new Date().toISOString(),
+        action: 'creation',
```

Comment:
For your information,  @SanKson7, @lunatiique, and @floporsch911, I have added an `action` field.

Indeed, if one day we add other events in the history, we will not be sure that the first event is the creation event (for documents created before the current commit).
By recording it as a creation from today, we won't have to modify every document in the database to update the structure.